src/Control/Eff/Cut.hs

```diff
@@ -65,7 +70,7 @@ cutfalse = throwExc CutFalse
 -- of its argument computation. When it encounteres a cutfalse request,
 -- it discards the remaining choicepoints.
 -- It completely handles CutFalse effects but not non-determinism.
-call :: Member Choose r => Eff (Exc CutFalse :> r) a -> Eff r a
+call :: forall r a . Member Choose r => Eff (Exc CutFalse :> r) a -> Eff r a
```

Comment:
@qnikst failing build. the type signature probably needs to be ``#ifdef``d as well